homeassistant/components/motioneye/camera.py

```diff
@@ -159,7 +159,7 @@ def __init__(
         self._motion_detection_enabled: bool = camera.get(KEY_MOTION_DETECTION, False)
 
         # motionEye cameras are always streaming or unavailable.
-        self.is_streaming = True
+        self._attr_is_streaming = True
 
         MotionEyeEntity.__init__(
```

Comment:
Calling parent constructors at the end seems like it adds possibility of side effects, like it did for the bug fixed in this PR.   @dermotduffy as an FYI in case this is something you think is worth fixing.

Reply:
Not sure how that is related to this PR.

Reply:
You updated motion eye tests in this PR changing state from `idle` to `streaming`. This already set `is_streaming = True` but the motion eye tests showed the state as `ide` which is really just showing two problems (a bug here, and a bug in the test). My impression is this happens because this calls the `MjpegCamera.__init__` which calls `Camera.__init__` which overwrote `is_streaming = False` in the constructor.  It is related to this PR because you fixed the bug and the tests were updated to show that the bug is fixed.  I'm not saying it needs to be fixed in this PR, i meant that as an FYI to improve it in a follow up, or even pre-factor.